src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py

```diff
@@ -79,28 +82,29 @@ def get_metadata(self) -> DeepDict:
             imaging_plane=imaging_plane_name,
             scan_line_rate=1 / float(xml_metadata["scanLinePeriod"]),
             rate=self.imaging_extractor.get_sampling_frequency(),
-            description=f"{self.imaging_purpose} imaging data ({indicator}) acquired from the Bruker Two-Photon Microscope",
-            unit="px",
+            description=f"{self.imaging_purpose} imaging data ({indicator})",
+            unit="voxels",
```

Comment:
I am not sure about this @alessandratrapani what do you think?

Reply:
Actually it should be n.a. as in the default value. My mistake, I misunderstood the meaning here. For reference see this [discussion](https://github.com/NeurodataWithoutBorders/nwb-schema/issues/460#:~:text=units%20of%20collected%20data%20%2D%20virtually%20no%20one%20knows%20it.%20At%20best%20you%20can%20express%20it%20in%20photon%20numbers%20(unitless)%20or%20W/pixel%20but%20the%20calibration%20is%20tricky%20and%20a%20very%20few%20labs%20could%20provide%20this%20information.)

Reply:
OK, changed, thanks!